src/main/middleware.ts

```diff
@@ -1,36 +1,20 @@
+import { StructuredLogHttpRequest } from '@nodescript/logger';
 import { Context } from 'koa';
 import { v4 as uuid } from 'uuid';
 
 import { ServerError } from './exception.js';
 
 export async function standardMiddleware(ctx: Context, next: () => Promise<any>) {
+    let error: any = undefined;
     const startedAt = Date.now();
     const requestId = ctx.headers['x-request-id'] || uuid();
-    const agent = ctx.header['user-agent'];
     ctx.state.requestId = requestId;
     try {
         await next();
-        ctx.logger.info(`HTTP request`, {
-            method: ctx.method,
-            url: ctx.url,
-            status: ctx.status,
-            took: Date.now() - startedAt,
-            agent,
-            requestId,
-        });
-    } catch (error: any) {
+    } catch (err: any) {
```

Comment:
Not sure if this can happen in `next()`, if it can, it's better to be safe than sorry

<img width="667" alt="image" src="https://github.com/ubio/node-framework/assets/64712584/9ab23e5d-bc10-4223-baa4-b2867b43eb6a">


```suggestion
    } catch (err: unkown) {
```